doc/source/index.rst

```diff
@@ -72,7 +72,8 @@ You can also install PyAEDT from Conda-Forge with this command:
 
 PyAEDT remains compatible with IronPython and can be still used in the AEDT Framework.
 
-PyAnsys
+
+PyANSYS
```

Comment:
There should be no instance where we use PyANSYS. Ansys is always to be lower case except in copyright statements.

Reply:
Thank you I've added to accept.txt